### PR TITLE
fix(#458): Support Deezer track URLs with a locale/country prefix

### LINKED ISSUE #458: Deezer: support track URLs which contain a locale/country prefix
E.g. https://www.deezer.com/fr/track/115996896 (instead of https://www.deezer.com/track/115996896)

Paste the URL in the search bar => you'll see that only the latter is supported.

### PR DESCRIPTION
**Pull request by has12zen**
Closes #458

## What does this PR do / solve?

adds support for searching deezer.com/fr/track:id urls

## How to test this PR?

url: http://www.deezer.com/fr/track/73414915
expected behavior: preview is loaded

### CODE REVIEW

**Comment by adrienjoly** [comment]
👋  Thanks again for your fixes!

Similarly to https://github.com/adrienjoly/playemjs/pull/43#issuecomment-849862529, I think that your code editor reformatted all the files, which makes the actual changes of your PR quite hard to find in the diff, unfortunately...

Unless you want to open a new PR while disabling the auto-formatting of your editor, I'm going to try to fix this in a few commits here.

File: docs/playemjs-fix-procedure.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ The procedure consists of the following steps:
 2. **Identify this issue in the ticket management system** (i.e. Github Issues), or create it. Share data from step 1. Leave a trace that someone experienced this problem, and state your intentions. (keeping a precise history can help fix a stealthy bug)
 3. **Make sure that there is a test in the [playemjs repo](https://github.com/adrienjoly/playemjs) that fails in a matching context** (cf data from step 1), or write that test and commit it with a message like "failing test for openwhyd issue #XXX".
 4. **Make the test pass by fixing playemjs**, then -- after making sure you did not break playemjs' other tests -- commit your fix with a message like "fixed openwhyd issue #XXX".
-5. **Make sure that this fix solves the problem on Openwhyd**: in your local `playemjs` directory, run `make compile`, then copy the `/dist/playemjs-*.js` files to the `/public/js` directory of your local Openwhyd repository, and test step 1's scenario.
+5. **Make sure that this fix solves the problem on Openwhyd**: in your local `playemjs` directory, run `make build`, then copy the `/dist/playemjs-*.js` files to the `/public/js` directory of your local Openwhyd repository, and test step 1's scenario.
```
Comment (adrienjoly) [inline docs/playemjs-fix-procedure.md:39]: 👍  good catch!

**Comment by adrienjoly** [comment]
FYI, I'll be back at this PR, after my day of work, this evening.

**Comment by has12zen** [comment]
```
  1) Visual Snapshots
       "before all" hook for "visitor":
     Error: APPLITOOLS_API_KEY env variable is missing. It is required to define this variable for Applitools visual tests to run successfully.
```
```
Warning: It looks like you are trying to record this run from a forked PR.

The 'Record Key' is missing. Your CI provider is likely not passing private environment variables to builds from forks.

These results will not be recorded.

This error will not alter the exit code.
```
maybe tests are failing because env variables are not being passed

**Review by adrienjoly — APPROVED** [review]
LGTM

Let's see if me approving your PR will let GitHub run the CI workflow with access to required secrets (such as the `APPLITOOLS_API_KEY` env variable)

**Pull request merged**